### PR TITLE
PUBLISH: 7/18 Migrate from polling to AWS metric streams

### PR DESCRIPTION
**Pull request by vinrelic**
Added details on the recent Migrate to AWS Cloudwatch Metric Streams Button. Please note that this change has to go live on July 14th to make sure it is in sync with the feature availability.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by vinrelic** [comment]
Added details on the recent Migrate to AWS Cloudwatch Metric Streams Button. Please note that this change has to go live on July 14th to make sure it is in sync with the feature availability.

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 1]
```diff
@@ -94,15 +94,20 @@ We recommend sending the data to a non-production New Relic account where you ca
...
-3. Repeat step 2 for any additional AWS region you want to monitor, since AWS CloudWatch requires one stream per region.
-4. Ensure metrics are received from all connected regions and namespaces. This may take several minutes.
-5. Disable all unnecessary polling integrations in the previous AWS provider account. [Remember that some integrations still need to be enabled](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream#integrations-not-replaced-streams) since they aren't fully replaced by metric streams.
+1. Replicate the namespaces from polling to metric streams by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS**. Click on the **Migrate to AWS Cloudwatch metric streams** button, then configure your AWS Metric Streams account. 
+2. Download the customized CloudFormation template. This template contains the preconfigured namespaces that are based on your existing polling configuration.
+3. Add your account details. This lets you get the migration complete easily.
+
+**Alternate option:**
```
Comment (akristen) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:105]: Hi @vinrelic -- when is this option a good option for users? What circumstance do we recommend the alternative migration?

Comment (vinrelic) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:105]: Alternative option is for customers who prefer not to use the CloudFormation template.

**Comment by vinrelic** [comment]
Team, can we delay the publishing of this document to Monday? Our engineering team is requesting for a day to get this deploying in production. Looking forward to your confirmation.

**Comment by zuluecho9** [comment]
@vinrelic realized this was, I think, meant to go out yesterday. Is that right? Can i publish this now?

**Pull request merged**